scripts/generate-wiki-markdown.js

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const path = require('path')
+const https = require('https')
+
+const repoPath = 'nuxt/nuxt.js'
+const pkgNamespace = '@nuxt'
+const branches = [
```

Comment:
Maybe we can move this to the top level `package.json` and using Package class to read?

Reply:
Same as above 🙃